content/docs/tasks/security/auth-sds/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Identity provision through SDS
+description: Shows how to enable SDS(secret discovery service) for istio identity provision.
+weight: 70
+keywords: [security,auth-sds]
+---
+
+This task shows how to enable [SDS(secret discovery service)](https://www.envoyproxy.io/docs/envoy/latest/configuration/secret#config-secret-discovery-service) for istio identity provision.
+
+Today the keys and certificates of Istio workloads are generated by Citadel and distributed to sidecars
+through secret-mount files, this approach has below side effects:
+* Traffic interruption:
+  When certificate rotation happens, envoy(pilot agent) needs to restart to pick up key/certificate update,
+  which causes traffic interruption.
+* Potential security issue:
+  Since key/certificate are file-mounted to sidecars, hackers could remote to sidecar container and steal key/certificate content.
+
+To solve the above issues, starting from 1.1 release, Istio supports identity provision through SDS(secret discovery service).
+
+Workload requests key/certificate from SDS server(node agent, which runs as per-node daemon set) using k8s service account JWT through
+SDS API, node agent generates private key and send CSR request to Citadel, Citadel verifies the JWT and signs the certificate,
+the key/certificate will eventually be sent back to workload sidecar through SDS server(node agent).
+Since key/certificate which are generated through SDS are stored in workload sidecar's memory, it's more secure than file mount;
+also the workload doesn't need to restart to pick up the certificate change when rotation happens.
+
+## Before you begin
+
+* Set up Istio by following the instructions using [Helm](/docs/setup/kubernetes/helm-install/) with SDS setup and global mutual TLS enabled:
+
+    {{< text bash >}}
+    $ cat install/kubernetes/namespace.yaml > istio-auth-sds.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-* >> istio-auth-sds.yaml
+    $ helm dep update --skip-refresh install/kubernetes/helm/istio
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml >> istio-auth-sds.yaml
+    $ kubectl create -f istio-auth-sds.yaml
+    {{< /text >}}
+
+## service-to-service mutual TLS using key/certificate provisioned through SDS
+
+Follow task in [authentication policy](/docs/tasks/security/authn-policy/) to setup test services.
```

Comment:
Follow the authentication policy task to...

Reply:
done.